sqlite/src/analyze.c

```diff
@@ -2154,6 +2154,7 @@ static int loadStatTbl(
     }
     pSpace = (tRowcnt*)&pIdx->aSample[nSample];
     pIdx->aAvgEq = pSpace; pSpace += nIdxCol;
+    pIdx->pTable->tabFlags |= TF_HasStat4;
```

Comment:
The patch isn't working as for 7.0 this function is defined inside 
#if !defined(SQLITE_BUILDING_FOR_COMDB2)
https://github.com/bloomberg/comdb2/blob/823e50ba381538a00f9a02ecf81475c4d113b00a/sqlite/src/analyze.c#L2083

even the loadStatTbl never gets called because of
https://github.com/bloomberg/comdb2/blob/823e50ba381538a00f9a02ecf81475c4d113b00a/sqlite/src/analyze.c#L2385

Reply:
Thanks, I missed a customization from the SQLite 3.35 branch PR.  Should be ok now.